src/screens/Asset/TradingActivity.tsx

```diff
@@ -60,6 +60,8 @@ const TradingActivityScreen = () => {
     );
   };
 
+  const getKeyExtractor = (item, index) => item.amm + index.toString();
```

Comment:
Repeated code, this should also be in a utils file from what it looks like

Reply:
Updated it!